### PR TITLE
[7.4 Only] LPS-122387 Call form view creation page on modal after save the new custom object

### PR DESCRIPTION
**Pull request by gabrielpramos**
story: [LPS-111511](https://issues.liferay.com/browse/LPS-111511)

### CODE REVIEW

**Comment by gabrielpramos** [comment]
needs rebase from [this pr](https://github.com/liferay-workflow/liferay-portal/pull/112)

**Comment by gabrielpramos** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by gabrielpramos** [comment]
ci:forward

**Comment by gabrielpramos** [comment]
ci:forward

File: modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js [hunk 1]
```diff
@@ -105,6 +110,7 @@ export default function DataAndViewsTab({
 	dispatch,
 	dispatchConfig,
 }) {
+	const {objectsPortletURL} = useContext(AppContext);
```
Comment (rafaprax) [inline modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js:113]: Hey @gabrielpramos can you move the `const {objectsPortletURL} = useContext(AppContext)` statement close to its use?

Thx

**Pull request closed without merging**